### PR TITLE
Request to add SAM payment channel icon

### PR DESCRIPTION
**Pull request by jimanx2**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

Related #456

### CODE REVIEW

**Comment by jimanx2** [comment]
rake test result:

image

**Comment by jimanx2** [comment]
Hello @tauthomas01 @larouxn. Requesting your kind help to review this.

**Review by larouxn — APPROVED** [review]
LGTM 🎩 
sam

**Comment by jimanx2** [comment]
Hi. Any updates on this? Can I check the ETA for release?

File: app/assets/images/payment_icons/sam.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...219-1.097h.379l.02.738.339-.738h.359l-.219 1.097h-.219l.179-.858-.379.858h-.219l-.02-.858-.179.858h-.26z"/></g></g><g><path d="M3.154 12.877c-.06.14-.259.239-.459.239H.222c-.199 0-.299-.12-.239-.239l.798-1.815c.06-.14.279-.239.459-.239h2.473c.199 0 .299.12.239.239l-.798 1.815z" fill="none"/><path fill="none" d="M-.017 10.823h4.048v2.313H-.017z"/></g><g><path d="M37.157 12.877c-.06.14-.259.239-.459.239h-2.473c-.199 0-.299-.12-.239-.239l.798-1.815c.06-.14.259-.239.459-.239h2.473c.199 0 .299.12.239.239l-.798 1.815z" fill="none"/><path fill="none" d="M33.986 10.823h4.048v2.313h-4.048z"/></g><g><path d="M20.125 4.103c-.06.14-.259.239-.459.239h-2.453c-.199 0-.299-.12-.239-.239l.798-1.815c.06-.14.259-.259.439-.259h2.473c.199 0 .299.12.239.239l-.798 1.835z" fill="none"/><path fill="none" d="M16.955 2.029h4.048v2.313h-4.048z"/></g><g><path d="M20.125 21.732c-.06.14-.259.239-.459.239h-2.453c-.199 0-.299-.12-.239-.239l.798-1.815c.06-.14.259-.239.459-.239h2.473c.199 0 .299.12.239.239l-.818 1.815z" fill="none"/><path fill="none" d="M16.955 19.658h4.048v2.313h-4.048z"/></g><path fill="none" d="M4.032 4.322h29.954v15.316H4.032z"/><path fill="none" d="M-.017 2.029h38.011v19.943H-.017z"/></g></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/sam.svg:1]: Can we remove these attributes `version="1.1" id="pi-Layer_1" x="0" y="0" xml:space="preserve"`? It doesn't look like we need it.

**Review by tauthomas01 — CHANGES_REQUESTED** [review]

File: app/assets/images/payment_icons/sam.svg (hunk 1 above)
Comment (jimanx2) [reply app/assets/images/payment_icons/sam.svg]: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 24"...`

removed as per your request

**Comment by tauthomas01** [comment]
Looks good. Can you rebase your branch off `master`? There's a conflict.

**Comment by jimanx2** [comment]
> Looks good. Can you rebase your branch off `master`? There's a conflict.

Give me a moment

**Comment by jimanx2** [comment]
Conflict resolved

**Review by tauthomas01 — APPROVED** [review]

**Pull request merged**